docs/SLACK_BOT.md

```diff
@@ -0,0 +1,20 @@
+## How to setup the Slack Bot?
+
+1. Create a [Slack App](https://api.slack.com/apps?new_app=1).
+2. Check your newly created app in [Dashboard](https://api.slack.com/apps/)
+3. Go back to Slack. Create a channel, give a name(ex: vacabot)
+4. In left side bar, choose *Incoming Webhooks* and *Activate Incoming Webhooks*.\
+   Scroll down to *Webhook URLs for Your Workspace* and add a new one.
+   ![Webhooks](./docs/vacabot-webhooks.png)
```

Comment:
The relative paths for images are wrong.
`./vacabot-webhooks.png`
The same goes for other images.

Reply:
Good catch. Just updated again